<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>M11DocumentDialog</class>
 <widget class="QDialog" name="M11DocumentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="lblStoreFrom">
       <property name="text">
        <string>Отправитель</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="CStoreComboBox" name="cmbStoreFrom">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblDocumentId">
       <property name="text">
        <string>Порядковый номер</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="edtDocumentId">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="edtHistoryNumber">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="lblHistoryNumber">
       <property name="text">
        <string>Номер истории болезни</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblUser">
       <property name="text">
        <string>Заявитель</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="CUserComboBox" name="cmbUser">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="lblDate">
       <property name="text">
        <string>Дата создания</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDateTimeEdit" name="edtDatetime">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="displayFormat">
        <string>dd.MM.yyyy HH:mm</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="lblNumber">
       <property name="text">
        <string>Номер</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="edtNumber">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabRequestDocumentItems">
      <attribute name="title">
       <string>Запрашиваемые товары</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="CItemListView" name="tblRequestDocumentItems"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabM11DocumentItems">
      <attribute name="title">
       <string>Отпускаемые товары</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="CItemListView" name="tblM11DocumentItems"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnSelectItems">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Подобрать товар</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFinalize">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Отработать</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CStoreComboBox</class>
   <extends>QComboBox</extends>
   <header>Pharmacy/ItemListComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CItemListView</class>
   <extends>QTableView</extends>
   <header>library/ItemListView.h</header>
  </customwidget>
  <customwidget>
   <class>CUserComboBox</class>
   <extends>QComboBox</extends>
   <header>Pharmacy/ItemListComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtDocumentId</tabstop>
  <tabstop>cmbStoreFrom</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>tblRequestDocumentItems</tabstop>
  <tabstop>btnSelectItems</tabstop>
  <tabstop>btnFinalize</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>tblM11DocumentItems</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>M11DocumentDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>M11DocumentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
